src/main/java/be/syntra/devshop/DevshopBack/models/SearchModel.java

```diff
@@ -17,9 +17,34 @@
     private BigDecimal priceHigh;
     private boolean sortAscendingName;
     private boolean sortAscendingPrice;
+    private boolean nameSortActive;
+    private boolean priceSortActive;
     private boolean archivedView;
     private boolean searchResultView;
     private boolean searchFailure;
     private boolean activeFilters;
     private String appliedFiltersHeader;
+    private Integer pageNumber;
```

Comment:
these aren't going to be persisted. Wouldn't you rather want a primitive int so that the default value = 0, instead of null?

Reply:
see next comment